src/lightbox/tp-lightbox.ts

```diff
@@ -25,8 +29,16 @@ export class TPLightboxElement extends HTMLElement {
 		// Initialize parent.
 		super();
 
+		// Initialize
+		this.dialogElement = this.querySelector( 'dialog' );
+
+		// Swipe settings
+		this.swipeThreshold = Number( this?.getAttribute( 'swipe-threshold' ) ?? '200' );
```

Comment:
@abhishekxix `this?` is redundant here - also, let's not do this here in the constructor - let's do this in the place where it is needed